app/serializers/service_provider_serializer.rb

```diff
@@ -48,7 +48,13 @@ def certs
   end
 
   def pkce
-    object.openid_connect_pkce?
+    if IdentityConfig.store.service_providers_with_nil_pkce.include?(object.issuer)
+      nil
```

Comment:
[suggestion] i'm wondering if this should remain `object.openid_connect_pkce?` since in theory that partner could decide to change it explicitly to pkce